specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -120,6 +128,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/checkQuotaAvailability": {
```

Comment:
>/subscriptions/{subscriptionId}/ [](start = 5, length = 32)

the RP manifest lists this as a tenant-level operation.  Which is correct?

Reply:
After some feedback I am requesting changes to the manifest to use location based routing instead.